lib/services/auth.services.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+import 'package:the_apple_sign_in/scope.dart';
+import 'package:the_apple_sign_in/the_apple_sign_in.dart';
+import '../screens/home/home.screen.dart';
+import 'database.services.dart';
+
+
+class AuthMethods {
```

Comment:
mica observatie dar m-a derutat putin :D 
clasa e AuthMethods si se afla in folderul services in fisierul auth.services.dart
voi mai aveti un fisier in folderul ala: auth.service.dart ( service la singular ), care contine clasa AuthService() - si ea deja se ocupa de partea de logat prin email si parola.

eu as zice sa mutati si logatul cu google / apple, si register tot acolo :)